src/config.ts

```diff
@@ -20,28 +20,58 @@ export interface ValidationConfig {
   readonly properties?: { [key: string]: any };
 }
 
+export enum SynthesisFormat {
+  'CDK8s' = 'cdk8s',
+  'HELM' = 'helm',
+}
+
+export enum HelmChartApiVersion {
+  'V1' = 'v1',
+  'V2' = 'v2',
+}
+
+export interface HelmSynthesis {
+  readonly chartApiVersion?: HelmChartApiVersion;
+  readonly chartVersion?: string;
+}
+
 export interface Config {
   readonly app?: string;
   readonly language?: Language;
   readonly output?: string;
   readonly imports?: string[];
   readonly pluginsDirectory?: string;
   readonly validations?: string | ValidationConfig[];
+  readonly format?: SynthesisFormat;
+  readonly helmSynthConfig?: HelmSynthesis;
```

Comment:
This makes the config file kind of cumbersome. How about we have:

```yaml
language: typescript
...
synth:
  format: helm
  chartVersion: 0.0.1
  chartApiVersion: v2
```

?

Reply:
Looks cleaner, thanks Eli 😄